rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/skip_rename_class_different_namespace.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace CompanyA;
+
+use CompanyB\Storage;
+
+class Bar
+{
+    /**
+     * @var Storage
+     */
+    protected $storage;
+
+    public function __construct(
+        Storage $storage
+    ) {
+        \Log::error('Some loggin');
```

Comment:
import \Log is expected, except you disable it https://github.com/rectorphp/rector-src/blob/b1583c7d8636aa32e6d5f7bdf37162e3e38a3907/packages/Config/RectorConfig.php#L84

Reply:
You can update to before -> after fixture that make `\Log` still imported while keeping docblock like this

https://github.com/rectorphp/rector-src/blob/main/rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/instanceof_import.php.inc